pwd_guess_unit.py

```diff
@@ -395,7 +395,7 @@ def test_accuracy(self):
         mock_model.train_on_batch = MagicMock(return_value = (0.5, 0.5))
         mock_model.test_on_batch = MagicMock(return_value = (0.5, 0.5))
         t.model = mock_model
-        self.assertEqual(0.5, t.train_model_generation())
+        self.assertEqual((0.5, False), t.train_model_generation())
```

Comment:
It would be good to add some tests for the early stopping behavior. I could imagine a test that checks that the state in the early_stopping function works.

Also, if tensorboad is turned off by default, then adding a test with it on would be good.